### PR TITLE
Changed line to review

### PR DESCRIPTION
**Pull request by ricconoel**
Updated comment and please review

### CODE REVIEW

**Comment by gelilio** [comment]
Testing comment

**Review by gelilio — COMMENTED** [review]
something review test

**Review by gelilio — CHANGES_REQUESTED** [review]
REJECT

File: main.py [hunk 1]
```diff
@@ -125,4 +125,4 @@ def main(request):
     answer_runner(start_str=start_str,end_str=end_str,start_date=start_date,end_date=end_date,table_name_suffix=table_name_suffix)
     comment_runner(start_str=start_str,end_str=end_str,start_date=start_date,end_date=end_date,table_name_suffix=table_name_suffix)
     query_runner(table_name_suffix=table_name_suffix)
-    #ricco is not here
+    #please review this line
```
Comment (ricconoel) [inline main.py]: change this line again

Comment (gelilio) [inline main.py]: Testing review

File: main.py [hunk 2]
```diff
@@ -125,4 +125,4 @@ def main(request):
     answer_runner(start_str=start_str,end_str=end_str,start_date=start_date,end_date=end_date,table_name_suffix=table_name_suffix)
     comment_runner(start_str=start_str,end_str=end_str,start_date=start_date,end_date=end_date,table_name_suffix=table_name_suffix)
     query_runner(table_name_suffix=table_name_suffix)
-    #ricco is not here
```
Comment (gelilio) [inline main.py:128]: Review Test

File: main.py (hunk 1 above)
Comment (ricconoel) [reply main.py]: Done

**Review by ricconoel — COMMENTED** [review]
LGTM

**Review by gelilio — APPROVED** [review]
LGTM, APPROVED

**Pull request merged**